src/libs/actions/Report.js

```diff
@@ -448,17 +448,35 @@ function reconnect(reportID) {
 }
 
 /**
- * Gets the older actions that have not been read yet.
- * Normally happens when you scroll up on a chat, and the actions have not been read yet.
+ * Retrieves the next set of report actions for the chat once we are nearing the end of what we are currently
+ * displaying.
  *
  * @param {String} reportID
- * @param {String} reportActionID
+ * @param {Object} reportActions
+ * @param {Boolean} isLoadingMoreReportActions
  */
-function readOldestAction(reportID, reportActionID) {
+function loadMoreActions(reportID, reportActions, isLoadingMoreReportActions) {
+    // Only fetch more if we are not already fetching so that we don't initiate duplicate requests.
+    if (isLoadingMoreReportActions) {
+        return;
+    }
+
+    const minSequenceNumber = _.chain(reportActions)
```

Comment:
This code is incorrect. We've refactored the code base and there should not be sequence numbers anymore.
We instead fetch using reportActionID. 

You shouldn't change this from reportActionID to reportActions since it can break the App. Instead at the place where you call loadMoreActions you should pass the current reportActionID as the second param and it will load actions before it.